using-mobile-extensions/adobe-analytics-edge/README.md

```diff
@@ -0,0 +1,199 @@
+# Adobe Analytics Edge
+
+{% hint style="warning" %}
+
+The AEP Analytics Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more.
+
+{% endhint %}
+
+## **Configure the Adobe Experience Platform Edge extension in** Experience Platform **Launch**
+
+1. In Experience Platform Launch, click the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Experience Platform Edge** extension, and click **Install**.
+3. Within the Edge Extension configuration page add your Edge configuration and environment settings. For more information on creating an Edge configuration for use with your mobile app, see [Experience Platform Setup](../../beta/experience-platform-extension/experience-platform-setup). Additionally, Analytics Edge extension setup steps are available in a tutorial at [Enable Analytics in Edge configuration.](../../beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension#enable-analytics-in-edge-configuration)
```

Comment:
should the links point to the new Edge extension docs?

Reply:
i'll update this